tests/test_compression_lr_multiplier.py

```diff
@@ -0,0 +1,268 @@
+def ref_and_target_configs(request) -> Tuple[NNCFConfig, NNCFConfig]:
+    ref_configs = [get_ref_config_fn() for get_ref_config_fn in request.param['get_orig_config_fns']]
+    ref_config = merge_configs(ref_configs, request.param['use_algo_list'])
+
+    target_configs = [add_multiplier_to_config(config, local_multiplier=multiplier)
+                      for config, multiplier in zip(ref_configs, request.param['multipliers'])]
+    target_config = merge_configs(target_configs, request.param['use_algo_list'])
+    target_config = add_multiplier_to_config(target_config, global_multiplier=request.param['global_multiplier'])
+
+    return ref_config, target_config
+
+
+def get_params_grouped_by_algorithms(model: nn.Module) -> Dict[str, Iterable[nn.Parameter]]:
+    cls_name_to_params = {}
+    modules = model.modules()
+    for module in modules:
+        params = module.parameters(recurse=False)
+        full_cls_name = '.'.join([module.__class__.__module__, module.__class__.__name__])
+        if full_cls_name not in cls_name_to_params:
+            cls_name_to_params[full_cls_name] = []
+        cls_name_to_params[full_cls_name].extend(params)
+
+    algo_name_to_params = {'regular': []}
+    for cls_name, params in cls_name_to_params.items():
+        params = [param for param in params if param.requires_grad]
+        for algo_name, algo_path in ALGO_NAME_TO_PATH_MAP.items():
+            if algo_path in cls_name:
+                if algo_name not in algo_name_to_params:
+                    algo_name_to_params[algo_name] = []
+                algo_name_to_params[algo_name].extend(params)
+                break
+        else:
+            algo_name_to_params['regular'].extend(params)
+
+    return algo_name_to_params
+
+
+def get_params_after_train_steps(config: NNCFConfig, num_steps: int = 1) -> Dict[str, Iterable[nn.Parameter]]:
+    with set_torch_seed():
+        model = LeNet()
+        model, compression_ctrl = create_compressed_model_and_algo_for_test(model, config)
+
+        for param in model.parameters():
+            if param.requires_grad:
+                nn.init.normal_(param)
+        optimizer = SGD(model.parameters(), lr=0.1)
+
+        for i in range(num_steps):
+            optimizer.zero_grad()
+            x = torch.rand(config['input_info']['sample_size'])
+            y = model(x)
+            loss = (y ** 2).sum()
+
+            loss.backward()
+            optimizer.step()
+
+    return get_params_grouped_by_algorithms(model)
+
+
+def test_if_algorithms_add_params(ref_and_target_configs: Tuple[NNCFConfig, NNCFConfig]):
+    base_config, config_with_multiplier = ref_and_target_configs
+
+    algo_to_params = get_params_after_train_steps(config_with_multiplier)
+    algo_names = get_multipliers_from_config(base_config).keys()
+
+    assert sorted(algo_to_params.keys()) == sorted(list(algo_names) + ['regular'])
+
+
+def test_how_multipliers_affect_grads(ref_and_target_configs: Tuple[NNCFConfig, NNCFConfig]):
```

Comment:
`test_grad_values_are_multiplied`

Reply:
@vanyalzr same for this test.

Reply:
same, I'd go with `test_grad_values_are_multiplied`

Reply:
Done.